### PR TITLE
Allow regular e2e workloads to be scheduled on SR-IOV nodes

### PR DESCRIPTION
**Pull request by brianmcarey**
/cc @dhiller 

Signed-off-by: Brian Carey <bcarey@redhat.com>

### CODE REVIEW

**Comment by brianmcarey** [comment]
Placing a hold on this until it is ready to go in

/hold

**Review by dhiller — APPROVED** [review]
/approve

Agreed, let's wait until @ormergi 's change lands in kubevirt

**Pull request merged**
